docs/api_reference/src/quantities.tex

```diff
@@ -146,121 +2593,397 @@
+                         std::@\stdconcept{constructible_from}@<std::common_type_t<T, U>, T> &&
+                         std::@\stdconcept{constructible_from}@<std::common_type_t<T, U>, U>;
 
-\begin{itemdecl}
-export template<typename T, quantity_character Ch>
-concept @\deflibconcept{representation_of}@ =
-  @\libconcept{representation}@<T> && ((Ch == quantity_character::scalar && is_scalar<T>) ||
+template<typename T, typename U = T>
+concept @\defexposconceptnc{ScalableNumber}@ = std::@\stdconcept{regular_invocable}@<std::multiplies<>, T, U> &&
+                         std::@\stdconcept{regular_invocable}@<std::divides<>, T, U>;
+
+template<typename T>
+concept @\defexposconceptnc{CastableNumber}@ =
+  @\exposconceptnc{CommonTypeWith}@<T, std::intmax_t> && @\exposconceptnc{ScalableNumber}@<std::common_type_t<T, std::intmax_t>>;
+
+template<typename T>
+concept @\defexposconceptnc{Scalable}@ =
+  @\exposconceptnc{CastableNumber}@<T> ||
+  (@\exposconceptnc{CastableNumber}@<@\exposidnc{actual-value-type-t}@<T>> &&
+   @\exposconceptnc{ScalableNumber}@<T, std::common_type_t<@\exposidnc{actual-value-type-t}@<T>, std::intmax_t>>);
+
+template<typename T>
+concept @\defexposconceptnc{WeaklyRegular}@ = std::@\stdconcept{copyable}@<T> && std::@\stdconcept{equality_comparable}@<T>;
+
+template<typename T>
+concept @\deflibconcept{Representation}@ = (is_scalar<T> || is_complex<T> || is_vector<T> || is_tensor<T>) &&
+                         @\exposconceptnc{WeaklyRegular}@<T> && @\exposconceptnc{Scalable}@<T>;
+
+template<typename T, quantity_character Ch>
+concept @\deflibconcept{RepresentationOf}@ =
+  @\libconcept{Representation}@<T> && ((Ch == quantity_character::scalar && is_scalar<T>) ||
+                        (Ch == quantity_character::complex && is_complex<T>) ||
                         (Ch == quantity_character::vector && is_vector<T>) ||
                         (Ch == quantity_character::tensor && is_tensor<T>));
 \end{itemdecl}
 
-% FIXME Despite the `some_` prefix, it doesn't conform to the convention
-%       `template_of(^std::remove_cvref_t<T>) == ^template_name`.
+\rSec1[qty.types]{Quantity types}
+
+\rSec2[qty.traits]{Traits}
+
+\rSec3[qty.val.traits]{Values}
+
+\begin{codeblock}
+namespace mp_units {
+
+template<typename Rep>
+struct @\libglobal{quantity_values}@ : std::chrono::duration_values<Rep> {
+  static constexpr Rep one() noexcept;
+};
+
+}
+\end{codeblock}
+
+\pnum
+The requirements on \tcode{std::chrono::duration_values<Rep>}\irefcpp{time.traits.duration.values}
+also apply to \tcode{quantity_values<Rep>}.
+
+\begin{itemdecl}
+static constexpr Rep one() noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+if constexpr (requires {
+                { std::chrono::duration_values<Rep>::one() } -> std::@\stdconcept{same_as}@<Rep>;
+              })
+  return std::chrono::duration_values<Rep>::one();
+else
```

Comment:
There is no such thing as `std::chrono::duration_values<Rep>::one()`. I do not think that we should check for that.

Reply:
The idea is for users who already specialize it
to be able to add `one` there
rather than on `quantity_values`.

Reply:
Yeah, I know, but I think that it is wrong to encourage users to do that.